src/System.CommandLine.Tests/GlobalOptionTests.cs

```diff
@@ -1,31 +1,13 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System.Threading.Tasks;
 using FluentAssertions;
 using Xunit;
 
 namespace System.CommandLine.Tests
 {
     public class GlobalOptionTests
     {
-        [Fact]
```

Comment:
Can you walk me through why this doesn't apply anymore? I'm not following that thread.

Reply:
This test was green for the wrong reason after we stopped performing this kind of validation up front and introduced `CommandLineConfiguration.ThrowIfInvalid`. I'll be adding some additional global option tests and validations in a future PR.